lib/daisy_ui_components/loading.ex

```diff
@@ -0,0 +1,54 @@
+defmodule DaisyUIComponents.Loading do
+  use DaisyUIComponents.Component
+
+  attr :class, :string, default: nil
+  attr :id, :string, required: true
```

Comment:
no need for id, we can let users reference in rest attribute like button does

Reply:
[addressing some comments](https://github.com/phcurado/daisy_ui_components/pull/31/commits/98ea99728317a9c5cbaba6ca4f060ea05235c413)

[testing and rest attr](https://github.com/phcurado/daisy_ui_components/pull/31/commits/aba655a6c7838976866ca8820c69e1832f266892)